notebooks/concepts/imviz-wcs-rotation.ipynb

```diff
@@ -0,0 +1,223 @@
+    "import astropy.units as u\n",
+    "from astropy.utils.data import download_file\n",
+    "\n",
+    "from astroquery.mast import Observations\n",
+    "\n",
+    "from jdaviz import Imviz"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8c63b7c3-cdf6-4c90-b978-1864fb4956f1",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cwd = './.'\n",
+    "\n",
+    "uris = [\n",
+    "    'mast:HST/product/j9dk11010_drz.fits',\n",
+    "    'mast:HST/product/j9dk11020_drz.fits',\n",
+    "    'mast:JWST/product/jw02731-o001_t017_nircam_clear-f200w_i2d.fits',\n",
+    "    'mast:jwst/product/jw02731-o001_t017_nircam_clear-f187n_i2d.fits'\n",
+    "]\n",
+    "\n",
+    "for uri in uris:\n",
+    "    path = cwd + uri.split('/')[-1]\n",
+    "    Observations.download_file(uri, local_path=path)\n",
+    "\n",
+    "tess_path = download_file(\n",
+    "    'https://mast.stsci.edu/api/v0.1/Download/file/' + \n",
+    "    '?uri=mast:TESS/product/tess2023107122157-s0064-3-2-0257-s_ffic.fits', \n",
+    "    cache=True\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "401af77b-907a-4dbc-bdb0-b60501f270e9",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "imviz = Imviz()\n",
+    "\n",
+    "paths = [\n",
+    "    [cwd + 'jw02731-o001_t017_nircam_clear-f200w_i2d.fits', 'JWST/NIRCam F200W'],\n",
+    "    [cwd + 'jw02731-o001_t017_nircam_clear-f187n_i2d.fits', 'JWST/NIRCam F187W'],\n",
+    "    [cwd + 'j9dk11010_drz.fits', 'HST/ACS F658N (1)'],\n",
+    "    [cwd + 'j9dk11020_drz.fits', 'HST/ACS F658N (2)'],\n",
+    "    [tess_path, 'TESS FFI']\n",
+    "]\n",
+    "load_order = [0, 1, 2, 3, 4]\n",
+    "\n",
+    "with imviz.batch_load():\n",
+    "    for path, data_label in np.array(paths)[load_order]:\n",
+    "        imviz.load_data(path, data_label=data_label)\n",
+    "\n",
+    "po = imviz.plugins['Plot Options']\n",
+    "po.image_color_mode = 'Monochromatic'\n",
+    "\n",
+    "colors = np.array(['r', '#40eb34', '#03dbfc', '#03dbfc', 'y'])[load_order]\n",
+    "vmins = np.array([1.2, 5, 0, 0, 0])[load_order]\n",
+    "vmaxes = np.array([8, 150, 3, 3, 3e4])[load_order]\n",
+    "alphas = np.array([1, 1, 1, 1, 0.7])[load_order]\n",
+    "\n",
+    "zip_items = zip(\n",
+    "    imviz.app.data_collection, colors, alphas, vmins, vmaxes\n",
+    ")\n",
+    "\n",
+    "for i, (data, color, alpha, vmin, vmax) in enumerate(zip_items):\n",
+    "    if not data.label.startswith(\"CCW\"):\n",
+    "        po.layer = data.label\n",
+    "        po.stretch_function = 'arcsinh'\n",
+    "        po.image_opacity = alpha\n",
+    "        po.image_contrast = 1.15\n",
+    "        po.image_color = color\n",
+    "        po.stretch_vmin = vmin\n",
+    "        po.stretch_vmax = vmax\n",
+    "\n",
+    "imviz.link_data(link_type='wcs')\n",
+    "imviz.show('sidecar:split-right', height=1200)\n",
```

Comment:
The fancy sidecar stuff does not work in Notebook 7. I run this and I don't see Imviz anywhere until I remove the arguments.

```suggestion
    "imviz.show()\n",
```